client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx

```diff
@@ -476,12 +476,14 @@ class DomainManagementNavigationEnhanced extends Component {
 	getDeleteDomain() {
 		const { domain, isLoadingPurchase, purchase, selectedSite, translate } = this.props;
 		const domainType = domain && domain.type;
+		const currentUserIsOwner = domain.currentUserIsOwner;
```

Comment:
Very minor observation: in `transfer-in-domain-type` you used object destructuring (`const { currentUserIsOwner } = domain;`) and here you used direct assignment 🙂 either is fine, but I guess we should try to keep consistency.

Reply:
Thanks!  Good point!